### PR TITLE
Add UIComponents overview to documentation

### PR DESCRIPTION
**Pull request by DominikBucher12**
# What this PR do:
- Adds UIComponents overview to docusaurus, similar to Controllers overview
# How to test it

 https://ios-uicomponents-overview.loca.lt

### CODE REVIEW

**Comment by b-onc** [comment]
I can't find the ngrok link 🙃

File: docusaurus/docs/iOS/ui-components/ui-components-overview.md [hunk 1]
```diff
@@ -1,3 +1,241 @@
...
+- [NavigationRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/NavigationRouter):
+A root class for all routes in the SDK.
+- [ChatChannelListRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/ChatChannelListRouter):
+A `NavigationRouter` subclass that handles navigation actions of `ChatChannelListVC`.
+- [AlertsRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/AlertsRouter):
+A `NavigationRouter` instance responsible for presenting alerts.
+- [ChatMessageListRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/ChatMessageListRouter):
+A `NavigationRouter` subclass used for navigating from message-list-based view controllers.
+
+## ChatChannelList
```
Comment (b-onc) [inline docusaurus/docs/iOS/ui-components/ui-components-overview.md:21]:
```suggestion
## ChatChannelListVC
```

File: docusaurus/docs/iOS/ui-components/ui-components-overview.md [hunk 2]
```diff
@@ -1,3 +1,241 @@
...
+### Types
+
+- [NavigationRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/NavigationRouter):
+A root class for all routes in the SDK.
+- [ChatChannelListRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/ChatChannelListRouter):
+A `NavigationRouter` subclass that handles navigation actions of `ChatChannelListVC`.
+- [AlertsRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/AlertsRouter):
+A `NavigationRouter` instance responsible for presenting alerts.
+- [ChatMessageListRouter](../ReferenceDocs/Sources/StreamChatUI/Navigation/ChatMessageListRouter):
+A `NavigationRouter` subclass used for navigating from message-list-based view controllers.
```
Comment (b-onc) [inline docusaurus/docs/iOS/ui-components/ui-components-overview.md:19]: Let's move this after Composer

**Review by b-onc — CHANGES_REQUESTED** [review]
Is this autogenerated? I was thinking something with more images.. Let's discuss in Slack

**Review by DominikBucher12 — COMMENTED** [review]
Added some images, restructured the documentation a bit and tried to give little more overview. Still in draft because I need to finish second half.

**Pull request closed without merging**